locales-src/asrouter.ftl

```diff
@@ -90,17 +90,41 @@ cfr-whatsnew-button =
   .label = What’s New
   .tooltiptext = What’s New
 
+cfr-whatsnew-fx70-hero-title = { -brand-short-name } now fights harder for your privacy
+cfr-whatsnew-fx70-hero-body = The latest update enhances the Tracking Protection feature and makes it easier than ever to create secure passwords for every site.
+
+cfr-whatsnew-fx70-notes-body = Protection reports, Secure Proxy for public Wifi, improved account controls, and many more safety and security tools are included in the latest update.
+
+cfr-whatsnew-tracking-protect-title = Protect yourself from trackers
+cfr-whatsnew-tracking-protect-body = { -brand-short-name } blocks many common social and cross-site trackers that follow what you do online.
+cfr-whatsnew-tracking-protect-link-text = View your protection report
+
 # This string is displayed before a large numeral that indicates the total number
 # of tracking elements blocked. Don’t add $blockedCount to
 # your localization, because it would result in the number showing twice.
-cfr-whatsnew-tracking-protections-title =
+cfr-whatsnew-tracking-blocked-title =
   { $blockedCount ->
     [one] Tracker blocked
    *[other] Trackers blocked
   }
-cfr-whatsnew-tracking-protections-subtitle = Since { DATETIME($earliestDate, month: "long", year: "numeric") }
+cfr-whatsnew-tracking-blocked-subtitle = Since { DATETIME($earliestDate, month: "long", year: "numeric") }
+cfr-whatsnew-tracking-blocked-link-text = View Protection Report
+
+cfr-whatsnew-lockwise-backup-title = Back up your passwords
+cfr-whatsnew-lockwise-backup-body = Now generate secure passwords you can access anywhere you sign in.
+cfr-whatsnew-lockwise-backup-link-text = Turn on backups
+
+cfr-whatsnew-lockwise-take-title = Take your passwords with you
+cfr-whatsnew-lockwise-take-body = The { -lockwise-brand-short-name } mobile app lets you securely access your backed up passwords from anywhere.
+cfr-whatsnew-lockwise-take-link-text = Get the app
 
 cfr-whatsnew-panel-header = What’s New
+cfr-whatsnew-dismiss-button = Dismiss
+
+# Variables:
+#  $version (String) - Firefox browser version
+cfr-whatsnew-firefox-notes-title = { -brand-short-name } Version { $version }
```

Comment:
This could just be a plain "{ -brand-short-name } Version 70" as otherwise the remote message would need to somehow specify args? `{string_id: "cfr_whatsnew-firefox-notes-title", args: { version: "70" } }` ?? @piatra I see #5267 just always passes in a common object for all strings. That's an interesting approach that could work too but perhaps we should name the args more explicitly if it's a shared "namespace", so all strings can use any args, `… { $firefoxVersion }` ?

Reply:
Yes I like this approach.

Reply:
Removing this particular string as we don't have "release notes" message anymore (but do have the "hero")